src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -131,7 +124,7 @@ function MoneyTemporaryForRefactorRequestParticipantsSelector({participants, onF
             isCategorizeOrShareAction ? 0 : undefined,
         );
 
-        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(debouncedSearchTerm, participants, chatOptions.recentReports, chatOptions.personalDetails, personalDetails, true);
+        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(debouncedSearchTerm, participants.map((participant) => ({...participant, reportID: participant.reportID ?? ''})), chatOptions.recentReports, chatOptions.personalDetails, personalDetails, true);
```

Comment:
why this map is added ?

Reply:
Because of type diff - Participant[] vs ReportUtils.OptionData[] 

ReportUtils.OptionData[] requires reportID with value